pkg/kubeadm/tlsutil/tlsutil.go

```diff
@@ -0,0 +1,133 @@
+// Copied from https://github.com/coreos/bootkube/blob/master/pkg/tlsutil/tlsutil.go
+// TODO vendor it properly
+// TODO also see whether some of these should just go into "pkg/util/certificates"
```

Comment:
Notes from Slack: I'm most in favor of replacing this with utility functions from cfssl, but moving it into `pkg/util/crypto` is an acceptable plan B. I'd really like to avoid writing yet another idiosyncratic stdlib wrapper.